$MirOS: ports/print/cups/patches/patch-config-scripts_cups-directories_m4,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- config-scripts/cups-directories.m4.orig	Wed Apr 26 19:52:04 2006
+++ config-scripts/cups-directories.m4	Tue May  9 17:39:25 2006
@@ -115,7 +115,7 @@ AC_ARG_WITH(rcdir, [  --with-rcdir      
 
 if test x$rcdir = x; then
 	case "$uname" in
-		FreeBSD* | OpenBSD*)
+		FreeBSD* | OpenBSD* | MirBSD* | ekkoBSD*)
 			# FreeBSD and OpenBSD
 			INITDIR=""
 			INITDDIR=""
